guides/updating/update-indy-sdk-to-askar.md

```diff
@@ -4,7 +4,7 @@ This documentation explains the process of migrating your Indy SDK wallet to [Ar
 
 :::danger
 
-While the migration script technically works on node.js, it is strongly advised not to use it, yet. The migration of issuer records (such as Schemas and Credential Definitions) is not implemented yet. When a credential definition is detected it will revert the migration process and no harm is done.
+While the migration script technically works on Node.JS, it is strongly advised not to use it, yet. The migration of issuer records (such as Schemas and Credential Definitions) is not implemented yet. When a credential definition is detected it will revert the migration process and no harm is done.
```

Comment:
I'll never get the correct spelling of Node.JS :').

Reply:
Ah appreantly it's "Node.js" (from their webstie). LOL I've been writing it wrong for a long time. Will update

Reply:
I have done all the spellings at some point and lately I've just stuck with `node.js` which apparently is incorrect. Oh well :)